wgengine/monitor/monitor.go

```diff
@@ -50,18 +57,20 @@ type Mon struct {
 	logf   logger.Logf
 	om     osMon // nil means not supported on this platform
 	change chan struct{}
-	stop   chan struct{}
-
-	mu       sync.Mutex // guards cbs
-	cbs      map[*callbackHandle]ChangeFunc
-	ifState  *interfaces.State
-	gwValid  bool // whether gw and gwSelfIP are valid (cached)x
-	gw       netaddr.IP
-	gwSelfIP netaddr.IP
+	stop   chan struct{} // closed on Stop
 
-	onceStart  sync.Once
+	mu         sync.Mutex // guards all following fields
+	cbs        map[*callbackHandle]ChangeFunc
+	ifState    *interfaces.State
+	gwValid    bool // whether gw and gwSelfIP are valid (cached)x
```

Comment:
What does "(cached)x" mean?

Reply:
the x is a typo. will fix.

can delete the cached too. doesn't add much.